src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -91,17 +97,38 @@ export class ColorPickerHexInput implements LoadableComponent {
    */
   @Prop() allowEmpty = false;
 
-  /** Specifies the size of the component. */
-  @Prop({ reflect: true }) scale: Scale = "m";
+  /**
+   * Specifies accessible label for the input field.
+   *
+   * @deprecated use `messages` instead
+   */
+  @Prop() hexLabel = "Hex";
 
   /**
-   * The Hex value.
+   * Messages are passed by parent component for accessible labels.
+   *
+   * @internal
    */
-  @Prop({ mutable: true, reflect: true }) value: string = normalizeHex(DEFAULT_COLOR.hex());
+  @Prop() messages: ColorPickerMessages;
 
   /** Specifies the Unicode numeral system used by the component for localization. */
   @Prop() numberingSystem?: NumberingSystem;
 
+  /**
+   * When true, the input will process and display hex characters for the alpha channel.
+   */
+  @Prop() opacityEnabled = false;
```

Comment:
Do we need to say `Enabled`? Would `opacity` be enough? Just wondering because I don't want to start adding a bunch of `xEnabled` properties. Usually, in HTML, the fact that the attribute is there or the property is true is the assumption that it is enabled.

Other possibilities

- `alphaChannel` - My fav.
- `displayAlpha`
- `processAlpha`
- `allowApha`

Reply:
I hear ya. Code-wise, it's treated as the flag to enable the feature, but it sort of fits with the existing props to hide UI parts. Would rephrasing the doc help here? If not, I can chew a bit more on the names. `opacity` doesn't indicate behavior to me and could be confused as the prop for getting/setting the opacity value.

Reply:
@driskull's top choice is the winner! :sparkles::skull::trophy::sparkles: